### ISSUE TITLE
api: provide fabric8 bindings

### ISSUE THREAD

**Issue by lburgazzoli** [apache/camel-k#2082]
We should contribute a [fabric8 Kubernetes Client extension](https://github.com/fabric8io/kubernetes-client/tree/master/extensions/)

**Comment by lburgazzoli** [comment]
Since the APIs will be tight to the fabric8 kubernetes-client, the extension should be hosted in the kubernetes-client like the [knative](https://github.com/fabric8io/kubernetes-client/tree/master/extensions/knative) one.

@nicolaferraro @davsclaus @oscerd what do you think ?

**Comment by oscerd** [comment]
+1

**Comment by davsclaus** [comment]
yeah sure that may allow users that come from fabric8 to find camel-k

**Comment by lburgazzoli** [comment]
Available here: https://github.com/fabric8io/kubernetes-client/tree/master/extensions/camel-k
